### ISSUE TITLE
Where is this actually hosted/served from?

### ISSUE THREAD

**Issue by jeremiak** [openchattanooga/cut-site#3]
I just made some changes per Tyler's request for some new Wufoo forms (336d42a0ebc27d363d486b3bffdcddb0e0171e90 and 9d9ce35a633d73b950f486fed6d0cc88501cde38) but it just occurred to me after trying to refresh the page that this is likely not set up for continuous deployment.

@seabre how can we get the latest master live so that the City can use their own Wufoo forms instead of Code for America's?

**Comment by seabre** [comment]
@jeremiak This appears to be hosted in an Amazon S3 bucket. Let me see if I have access to that.

**Comment by seabre** [comment]
@jeremiak Looks like I don't. Who worked with you on this?

**Comment by jeremiak** [comment]
Daniel Ryan I believe. Do we have DNS access? Maybe just host it off of github pages since its a flat site?

cc @seabre

**Comment by jeremiak** [comment]
Hey @seabre I just got an alert that somebody new signed up for the CUT group. `master` on this repo actually has a different form that will let the brigade and Tyler get notified instead of me, the problem is that we don't have this deployed.

See above comment. Can we change the DNS so that it points at the GH repo and we can just leverage gh-pages for the site?

**Comment by aplannersguide** [comment]
@seabre Let me know if there is anything I can do to help.

**Comment by aplannersguide** [comment]
@seabre how difficult it be to move this site to github pages?

**Comment by seabre** [comment]
Not hard at all. I'm not sure I have access to DNS stuff anymore. But as far as serving the site from GitHub pages, that is not a problem.

**Comment by aplannersguide** [comment]
Yeah, I'm not sure about where we left the DNS stuff for the open chattanooga site....that said, I'm thinking we could just use city DNS resources for the redirect....something like tester.chattanooga.gov. @seabre Any chance you could do a quick (5 minute) hangout today to walk me through the steps to move this to github pages?

**Comment by seabre** [comment]
@aplannersguide yeah, just let me know. I might have my head down working on something but I should be able to step away for 5 minutes.
